docs/ja/docs/tutorial/path-operation-configuration.md

```diff
@@ -0,0 +1,97 @@
+# Path Operationの設定
+
+*path operationデコレータ*を設定するためのパラメータがいくつかあります。
+
+!!! warning "注意"
+    これらのパラメータは*path operation関数*ではなく、*path operationデコレータ*に直接渡されることに注意してください。
+
+## レスポンスステータスコード
+
+*path operation*のレスポンスで使用する（HTTP）`status_code`を定義することができます。
+
+`404`のように`int`のコードを直接渡すことができます。
+
+しかし、それぞれの番号コードが何のためのものか覚えていない場合は、`status`のショートカット定数を使用することができます:
+
+```Python hl_lines="3  17"
+{!../../../docs_src/path_operation_configuration/tutorial001.py!}
+```
+
+そのステータスコードはレスポンスで使用され、OpenAPIスキーマに追加されます。
+
+!!! note "技術詳細"
+    また、`from starlette import status`を使用することもできます。
+
+    **FastAPI** は開発者の利便性を考慮して、`fastapi.status`と同じ`starlette.status`を提供しています。しかし、これはStarletteから直接提供されています。
+
+## タグ
+
+`tags`パラメータを`str`の`list`（通常は１つの`str`）と一緒に渡すと、*path operation*にタグを追加できます:
+
+```Python hl_lines="17 22 27"
+{!../../../docs_src/path_operation_configuration/tutorial002.py!}
+```
+
+これらはOpenAPIスキーマに追加され、自動ドキュメントのインターフェースで使用されます:
+
+<img src="https://fastapi.tiangolo.com/img/tutorial/path-operation-configuration/image01.png">
+
+## 概要と説明
+
+`summary`と`description`を追加できます:
+
+```Python hl_lines="20 21"
+{!../../../docs_src/path_operation_configuration/tutorial003.py!}
+```
+
+## docstringを用いた説明
+
+説明文は長くて複数行におよぶ傾向があるので、関数<abbr title="ドキュメントに使用される関数内の最初の式（変数に代入されていない）としての複数行の文字列">docstring</abbr>内に*path operation*の説明文を宣言できます。すると、**FastAPI** は説明文を読み込んでくれます。
+
+docstringに<a href="https://en.wikipedia.org/wiki/Markdown" class="external-link" target="_blank">Markdown</a>を記述すれば、正しく解釈されて表示されます。（docstringのインデントを考慮して）
+
+```Python hl_lines="19 20 21 22 23 24 25 26 27"
```

Comment:
```suggestion
```Python hl_lines="19-27"
```
current syntax

Reply:
oh, I see that the writing style of hl_lines has improved while I've been away for a bit.
 I'll fix it.